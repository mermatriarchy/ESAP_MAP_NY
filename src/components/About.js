import React from 'react';


export default function About() {
  return (
    <div>
      Info goes here.
    </div>
  );
}
