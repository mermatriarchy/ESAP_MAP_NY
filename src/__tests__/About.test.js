import React from 'react';
import renderer from 'react-test-renderer';

import About from '../components/About';

it('correctly renders About component', () => {  
    const AboutComponent = renderer.create(<About />).toJSON();
    expect(AboutComponent).toMatchSnapshot();
});