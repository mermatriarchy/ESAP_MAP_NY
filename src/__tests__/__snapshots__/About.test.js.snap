// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`correctly renders About component 1`] = `
<div>
  Info goes here.
</div>
`;
